### DISCUSSION TITLE
Outlook 2016

### DISCUSSION THREAD

**Discussion by JuliaJin05** [cmu-sei/GHOSTS#233] [Q&A]
How does GHOSTS determine which version of Outlook it is going to open. I am having trouble because every time I manually run ghosts.exe it opens the office365 version, but when I use this orchestration script [https://gitlab.cylab.be/cylab/ghosts-python/-/tree/master](url), it opens up a 2016 version. The orchestration script basically just lets you manually manage GHOSTS from one computer, so we can run GhostRunner.py and it will startup GHOSTS on every client.

**Comment by sei-dupdyke** [comment]
We use the redemption library, which _should_ pick up the default install. Are you saying that you want to use 365 or 2016 version?

**Reply by JuliaJin05** [reply]
I want to use the 365 version. I never even installed a 2016 office, but it keeps popping up when I try to run it with the orchestration script.

**Comment by sei-dupdyke** [comment]
Hmm, is the 365 version somehow set with the OS as the default handler for "mailto" links and similar? Here is a link that shows how to view these settings in Win11: https://www.howtogeek.com/746203/how-to-set-your-default-apps-on-windows-11/.

P.S. While I know GHOSTS will work with 2016, I'm not sure about performance with the 365 version.

**Reply by JuliaJin05** [reply]
> Hmm, is the 365 version somehow set with the OS as the default handler for "mailto" links and similar? Here is a link that shows how to view these settings in Win11: https://www.howtogeek.com/746203/how-to-set-your-default-apps-on-windows-11/.
> 
> 
> 
> P.S. While I know GHOSTS will work with 2016, I'm not sure about performance with the 365 version.

Yeah, I made sure to change the default app to Outlook
